.vscode/settings.json

```diff
@@ -69,5 +69,6 @@
     "wsol",
     "WSOLATA"
   ],
-  "editor.formatOnSave": true
+  "editor.formatOnSave": true,
+  "rust-analyzer.linkedProjects": ["./programs/uxd/Cargo.toml"]
```

Comment:
nit: i think we can remove this?

Reply:
Yhea not sure what's used for, we can

Reply:
I'm not sure what you mean, what commands are you running ?

If this is useful we can keep it tho, not a big deal!